CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.1-dev
+
+* Require Dart 2.18.0
```

Comment:
We haven't published `1.2.0` yet! Let's add this to the 1.2.0 section and plan to publish.